mirgecom/simutil.py

```diff
@@ -434,6 +434,16 @@ def create_parallel_grid(comm, generate_grid):
     return generate_and_distribute_mesh(comm=comm, generate_mesh=generate_grid)
 
 
+def configurate(config_key, config_object=None, default_value=None):
+    """Return a configured item from a configuration object."""
+    if config_object is not None:
+        d = config_object if isinstance(config_object, dict) else\
+            config_object.__dict__
+        if config_key in d:
+            return d[config_key]
+        return default_value
```

Comment:
This needs to be at the function level:
```suggestion
    return default_value
```